src/middlewares/validateUser.js

```diff
@@ -65,3 +65,50 @@ export const validateLogin = [
     .withMessage('Please input a password'),
   (req, res, next) => handleErrors(req, res, next),
 ];
+
```

Comment:
I am not sure why you did this. We already validated the user, why are you validating profile when profile is just the user details we already have?

Reply:
I didn't want to add confirm-password for when the user changes their password, but I see what you mean

Reply:
I've edited it to use the user validation

Reply:
A user still needs to confirm their password